tests/test_match.py

```diff
@@ -385,7 +385,7 @@ def test_known_fusions(self, conn, known_variant, related_variants):
             assert variant in names
 
     def test_known_fusion_single_gene_no_match(self, conn):
-        known = '(BCR,?):fusion(e.13,e.?)'
+        known = '(TERT,?):fusion(e.1,e.?)'
```

Comment:
Ah ok, I had a question about this in my PR: https://github.com/bcgsc/pori_graphkb_python/pull/66/files#r956280979

I think this is a reasonable change. I just pointed out that it's possible the test will break again depending on what gets uploaded to GraphKB.

Reply:
I agree, but I think we can deal with that if it comes up.

Right now, I think we first just need to get to something that passes these tests, so we can focus on the missense/nonsense category matching test.